transformations/word_stemmer/test.json

```diff
@@ -0,0 +1,50 @@
+{
+    "type": "simple_word_stemmer",
+    "test_cases": [
+      {
+        "class": "SimpleWordStemmer",
+        "inputs": {
+          "sentence": "Where did he learn to dance like that?"
+        },
+        "outputs": [{
+           "sentence": "where did he learn to danc like that?"
+        }]
+      },
+      {
+        "class": "SimpleWordStemmer",
+        "inputs": {
+          "sentence": "His eyes were dancing with humor."
+        },
+        "outputs": [{
+          "sentence": "hi eye were danc with humor."
+        }]
+      },
+      {
+        "class": "SimpleWordStemmer",
+        "inputs": {
+          "sentence": "She shook her head and danced away"
+        },
+        "outputs": [{
+          "sentence": "she shook her head and danc away"
```

Comment:
If you could maintain the casing in the output, that might help make the transformation specific?